CMakeLists.txt

```diff
@@ -21,6 +21,32 @@ if(DEFINED ENV{NANO_SEC})
   add_definitions(-DBOOST_DATE_TIME_POSIX_TIME_STD_CONFIG)
 endif()
 
+
+set(REGEX_EN)
+if(REGEX_EN MATCHES "HS")
+  find_path(HYPERSCAN_INCLUDE_DIR NAMES hs/hs.h)
+  find_library(HYPERSCAN_LIB1 NAMES hs)
+  find_library(HYPERSCAN_LIB2 NAMES hs_runtime)
```

Comment:
why not using find_package( ... QUIET ) --> if ( ...FOUND)

Reply:
find_package is not working for re2 and hyperscan in cmake so I used find_path and find_library